### PR TITLE
feat: Onboard DevConf.US 2021 cluster

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/operate-first/support/issues/368

We decided to spin up a new cluster for DevConf.US workshop. This PR onboards the cluster to Operate First once available.

### CODE REVIEW

File: odh-manifests/devconfus2021/jupyterhub/base/jupyterhub-singleuser-profiles-sizes-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            memory: 32Gi
+            cpu: 4
+      - name: Extra Large - Memory intensive
+        resources:
+          requests:
+            memory: 16Gi
+            cpu: 100m
+          limits:
+            memory: 48Gi
+            cpu: 4
```
Comment (tumido) [inline odh-manifests/devconfus2021/jupyterhub/base/jupyterhub-singleuser-profiles-sizes-configmap.yaml:41]: @pacospace do you have a size preference? Let's leave just a single one in there for  workshoppers :slightly_smiling_face:

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Comment by 4n4nd** [comment]
pre-commit was tested locally by @tumido. Merging :+1:

**Pull request merged**